### DISCUSSION TITLE
老是断联重试

### DISCUSSION THREAD

**Discussion by 17681093461** [natfrp/wiki#478] [使用问题]
我反向代理 请求接口上传完图片 然后就直接返回503 此情况可稳定复现只要上传图片就会出现重连
<img width="1160" alt="image" src="https://github.com/natfrp/wiki/assets/40077325/4bee8840-8fad-42b1-acb4-f3312e43384a">

**Comment by fakeboboliu** [comment]
请将详细信息（出现问题的隧道，使用的网站程序等）发送到工单邮箱，我们进行一下检查：
support@natfrp.cloud
